x-pack/test/defend_workflows_cypress/runner.ts

```diff
@@ -58,10 +60,16 @@ export async function DefendWorkflowsCypressVisualTestRunner(context: FtrProvide
 }
 
 export async function DefendWorkflowsCypressEndpointTestRunner(context: FtrProviderContext) {
-  await withFleetAgent(context, () => startDefendWorkflowsCypress(context, 'dw:endpoint:open'));
+  await withFleetAgent(context, (runnerEnv) =>
+    startDefendWorkflowsCypress(context, 'dw:endpoint:open', runnerEnv)
+  );
 }
 
-function startDefendWorkflowsCypress(context: FtrProviderContext, cypressCommand: string) {
+function startDefendWorkflowsCypress(
+  context: FtrProviderContext,
```

Comment:
Awesome. This was going to be one of my next things to look at - to pass `FtrProviderContext` to our tests for usage.